src/BlockTypes/Checkout.php

```diff
@@ -77,7 +77,9 @@ protected function render( $attributes, $content ) {
 		wp_dequeue_style( 'select2' );
 
 		// If the content is empty, we may have transformed from an older checkout block. Insert the default list of blocks.
-		$is_empty = strstr( $content, '<div class="wp-block-woocommerce-checkout is-loading"></div>' );
+		$regex_for_empty_block = '/<div class="[a-zA-Z0-9_\- ]*wp-block-woocommerce-checkout[a-zA-Z0-9_\- ]*"><\/div>/mi';
```

Comment:
This would match any empty div that has the:
- Must have a class attribute.
- Class attribute must have `wp-block-woocommerce-checkout`.
- Doesn't matter what other classes are other.
- Can only havea  class attribute, no other attributes are supported here, this is not an issue [according to this code.](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/release/5.9.0/assets/js/blocks/cart-checkout/checkout/index.js#L76-L82)
- `[a-zA-Z0-9_\- ]*` loosely matches a className, but isn't strict, it would support invalid classNames, but that's fine.